### PR TITLE
Proof reading 0:30 to 0:45 of Ranking Poker Hands, featuring Camille Baldock

### PR DESCRIPTION
**Pull request by tuzz**

### CODE REVIEW

File: 2/camille-baldock-captions.srt [hunk 1]
```diff
@@ -2166,7 +2166,7 @@ And for each of those cards I have Pips.
 
 498
 00:33:08,480 --> 00:33:11,920
-I can just inaudible
+I can just .....
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt:2169]: That's a lot of dots! I'd prefer to see `I can just...` - that is, no space then 3 dots. I'll update our style-guide to reflect this.

File: 2/camille-baldock-captions.srt [hunk 2]
```diff
@@ -2406,7 +2408,7 @@ be an array of those cards.
 
 555
 00:36:39,140 --> 00:36:44,560
-CB: I'm just wondering inaudible which is
+CB: I'm just wondering .....
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt:2411]: Again, let's do `wondering...`

File: 2/camille-baldock-captions.srt [hunk 3]
```diff
@@ -2789,7 +2791,7 @@ Shameless green ....
 
 647
 00:42:34,480 --> 00:42:39,290
-so if I have 3 then it becomes 3 of a kind
+so if I have 3 then it becomes :three_of_a_kind
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt:2794]: Looks good!

**Comment by nelstrom** [comment]
Great work @tuzz, thanks!

**Comment by nelstrom** [comment]
re: the elipses, I made those comments as notes-to-myself. Don't bother changing those yourself. I'll go through the entire document after merging all proofreader's PRs and make the elipses consistent.

**Comment by tuzz** [comment]
Good spot with the elipses. There's no real consistency across the whole document so you may need to work your Vim magic to standardise it. =)

**Pull request merged**